vignettes/scholar.Rmd

```diff
@@ -33,20 +34,23 @@ theme_set(theme_minimal())
 
 ```{r}
 ## Define the id for Richard Feynman
-id <- 'B7vSqZsAAAAJ'
+id <- 'MlMKkPUAAAAJ'
 
```

Comment:
It looks like the id is now for the psychologist James Gross not Feynman https://scholar.google.co.uk/citations?hl=en&user=MlMKkPUAAAAJ&sortby=pubdate&view_op=list_works